src/Microsoft.DiaSymReader/Reader/ISymUnmanagedBinder2.cs

```diff
@@ -7,12 +7,13 @@
 
 namespace Microsoft.DiaSymReader
 {
-    [ComImport]
     [Guid("ACCEE350-89AF-4ccb-8B40-1C2C4C6F9434")]
     [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
     [ComVisible(false)]
-    public interface ISymUnmanagedBinder2 : ISymUnmanagedBinder
+    [GeneratedWhenPossibleComInterface]
+    public partial interface ISymUnmanagedBinder2 : ISymUnmanagedBinder
     {
+#if NETSTANDARD2_0
```

Comment:
Are there any issues with these methods in NET9 build?

Reply:
Historically COM interop requires redefinition of inherited methods so the vtables are properly created. For the COM source generator in .NET 8+, the system respects "inheritance" from other COM interfaces. This means there is no need to redefine them.

Reply:
I see, could you please add a comment to the code?

Reply:
Sure. I assume in all the places where they are commented out?